torchdynamo_poc/utils.py

```diff
@@ -80,6 +82,8 @@ def torch_mlir_compiler(fx_graph: torch.fx.GraphModule,
         return fx_graph
 
     was_unwrapped = _unwrap_single_tuple_return(fx_graph)
+    fx_graph = make_fx(fx_graph)(*example_inputs)
+    strip_overloads(fx_graph)
```

Comment:
What are overloads and why do we need to strip them here?

(No need to document this if it's obvious.  Just trying to get up to speed.)